### DISCUSSION TITLE
Add unit testing and Continuous Integration

### DISCUSSION THREAD

**Discussion by Coder206** [Stellarium/stellarium#25] [Q&A] [answered]
Welcome to GitHub, thrilled the project is hosted here now!

I was looking over the readme and found no information about unit testing in the software. Do unit tests exist for Stellarium?

Also, it may be beneficial to setup a CI (Continuous Integration) like TravisCi or CircleCI, that way each PR is tested before merging

**Comment by cardinot** [comment, marked as answer]
Hi @Coder206,
thanks for your comments, and welcome to github too :stuck_out_tongue_closed_eyes:

We have been using TravisCi over the last 3 years. If you check out the PRs, you will see that all of them are checked... 
https://travis-ci.org/Stellarium/stellarium

Re. unit tests, yes, we have a few at 'stellarium/src/tests'... but we'd be happy to have more, feel free to contribute if you like.

**Comment by Coder206** [comment]
@cardinot My bad for the CI, it looked like GH's default "tests".

As for the unit tests, I think it would be beneficial mentioning this in the readme.
